python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -143,7 +143,7 @@ class ForcedPhotCcdConfig(ForcedPhotImageConfig):
     doApplyUberCal = lsst.pex.config.Field(
         dtype=bool,
         doc="Apply meas_mosaic ubercal results to input calexps?",
-        default=False
+        default=True
```

Comment:
Is this going to work for any cameras other than HSC where we have used meas_mosaic?